opentracing-api/pom.xml

```diff
@@ -31,5 +31,15 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
+        <mockito.version>1.10.19</mockito.version>
```

Comment:
nit-pick: don't use properties if there's no benefit in doing so.

if the version number is re-used over a number of artifacts there's a benefit.
here i'm unaware of any benefit.